.github/workflows/ci.yml

```diff
@@ -24,6 +24,17 @@ jobs:
           name: dist
           path: dist/index.js
 
+  lint:
```

Comment:
I like having a separate `lint` job, but this does duplicate the `yarn install` step. `build` and `lint` run concurrently, so it shouldn't add any time, but is extra work.

Alternatives would be to just add `yarn lint` as a step in `build`, or to do something more complicated to cache a `install` between the two.